manager/apis/app/v1beta1/helm_values.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1beta1
+
+// HelmValues are the values passed to modules during orchestration of the data plane
+type HelmValues struct {
```

Comment:
The parallel file in v1alpha1 is not used.

Reply:
OK.

Reply:
@shlomitk1 , maybe will be better to store the helm values in some other place. They are not part of K8s API.

Reply:
They were in other place but we wanted to make them visible.

Reply:
I suggest moving them to `pkg/helm`